packages/schema-extract/README.md

```diff
@@ -0,0 +1,79 @@
+# Schema Extract
+
+
+Schema extract transforms code files into [JSON-Schema](https://json-schema.org/) schemas. It is composed of two parts, the **schema-extract** and **linker**.
+
```

Comment:
Maybe hoist the basic linker usage, that utilizes the default ts transformer here. I would add an example to show input, command, and output